templates/etc/sensu/uchiwa.json.erb

```diff
@@ -20,7 +20,7 @@
     "port": <%= @port %>,
     "user": "<%= @user %>",
     "pass": "<%= @pass %>",
-    "refresh": <%= @refresh %><%= ',' if @users.size > 0 %>
+    "refresh": <%= @refresh %><%= ',' if @users.size > 0 or @auth.size == 2 %>
```

Comment:
This feels strange. I think eventually we need to move to deploying json files directly with ruby instead of writing out an erb